src/dash-table/syntax-tree/SingleColumnSyntaxTree.ts

```diff
@@ -1,9 +1,15 @@
+import * as R from 'ramda';
```

Comment:
That's a fair question. For this project both are equivalent and the tree-shaking will happen because of https://github.com/plotly/dash-table/blob/dev/tsconfig.base.json#L4 - otherwise this would actually be invalid usage in more recent version of ramda.